This is an experimental fix to chromium start failure.
Authored by: Camber Huang<camber@aosc.io>

---

--- a/chrome/browser/privacy_budget/identifiability_study_state.cc	2022-03-29 08:17:00.000000000 +0800
+++ b/chrome/browser/privacy_budget/identifiability_study_state.patched.cc	2022-03-30 22:14:07.998973020 +0800
@@ -56,13 +56,13 @@
       generation_(GetStudyGenerationFromFieldTrial()),
       active_surface_budget_(settings_.surface_budget()),
       random_offset_generator_(
-          settings_.expected_surface_count() > 0
+          settings_.expected_surface_count() > 9
               ? settings_.expected_surface_count()
               // If settings_.expected_surface_count() is 0 then the study is
               // disabled. The random offset generator will not be used.
               // However, `MesaDistribution` needs a `pivot_point` parameter
               // bigger than 0.
-              : 1,
+              : 10,
           kMesaDistributionRatio) {
   InitializeGlobalStudySettings();
   InitFromPrefs();
